server/user_data.py

```diff
@@ -8,18 +8,19 @@
 
 
 def get_character_list(user_id):
-    query = Character.get(Character.user_id == user_id)
+    query = Character.query().where(Character.user_id == user_id)
     return list(query.run())
 
 
 @cachetools.cached(cachetools.LRUCache(maxsize=1000))
 def get_character_data_list(user_id):
     character_dict = {}
     for character in get_character_list(user_id):
-        character_dict[character.character_id] = {
+        corp_id = character.corporation_id
```

Comment:
it was generating errors due to no corp! Perhaps it's a system char

Reply:
Can you revert the change, empty your database and debug this again? It really should not be, there might have been old Character entities left over when you got the error. As the code stands every new Character object is assigned a corporation id [[link](https://github.com/wgilpin/recruitment-ascee/blob/73435f9be277ba0131a87463b51e95063fe54a02/server/database.py#L340)].

Reply:
This code won't necessarily break anything, but it will mask whatever underlying bug is leading do corporation_id being undefined for some characters, if that bug exists.

Reply:
ok then I'll add a try catch and log it, then we can see if it happens